package.json

```diff
@@ -32,12 +49,13 @@
   "license": "MIT",
   "devDependencies": {
     "add-event-handler": "^1.0.0",
-    "babel": "^5.8.21",
```

Comment:
Now there is actually a use for the babel cli, so we have `babel-cli` installed.